osv/ecosystems_test.py

```diff
@@ -122,19 +130,52 @@ def test_debian(self, requests_mock: mock.MagicMock):
         ecosystem.enumerate_versions('nginx', '0', '<end-of-life>'), [])
     # Test Ecosystem remover
     ecosystem = ecosystems.get('Debian:10')
+
+    # Called 2 times so far, once for nginx, once for blender.
+    # Calls to the same ecosystem should be cached
+    self.assertEqual(general_requests_mock.call_count, 2)
     # '0' as introduced version also tests the get_first_package_version func
     versions = ecosystem.enumerate_versions('cyrus-sasl2', '0', None)
-    self.assertEqual(requests_mock.call_count, 2)
+    self.assertEqual(first_ver_requests_mock.call_count, 2)
+    self.assertEqual(general_requests_mock.call_count, 3)
+
     self.assertIn('2.1.27+dfsg-1+deb10u1', versions)
     self.assertNotIn('2.1.27~101-g0780600+dfsg-3+deb9u1', versions)
     self.assertNotIn('2.1.27~101-g0780600+dfsg-3+deb9u2', versions)
 
     with self.assertRaises(ecosystems.EnumerateError):
       ecosystem.next_version('doesnotexist123456', '1')
 
+    self.assertEqual(general_requests_mock.call_count, 4)
+
     # This should now only call the cache, and not requests.get
     ecosystem.enumerate_versions('cyrus-sasl2', '0', None)
-    self.assertEqual(requests_mock.call_count, 2)
+    self.assertEqual(first_ver_requests_mock.call_count, 2)
+    self.assertEqual(general_requests_mock.call_count, 4)
+    ecosystems.set_cache(None)
+
+  @mock.patch(
+      'osv.ensure_updated_checkout', side_effect=osv.ensure_updated_checkout)
+  def test_alpine(self, ensure_updated_checkout_mock: mock.MagicMock):
+    """Test Alpine"""
```

Comment:
Please improve this docstring to maybe include a human-readable summary of the test cases?